test/unit-test/mock/IAuthenticatorMock.hpp

```diff
@@ -20,8 +20,8 @@
 
 MOCK_BASE_CLASS( IAuthenticatorMock, IAuthenticator )
```

Comment:
What is the meaning of/the reason for this changes?
(Mabye it is obvious, but I am not familiar with our unittest setup)

Reply:
>What is the meaning of/the reason for this changes?

kuksa.val use [Boost::turtle](http://turtle.sourceforge.net/turtle/reference.html#turtle.reference.creation.class) create the mock class in unit test.
This change make sync with the IAuthenticator interface.